packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -51,72 +65,126 @@ export default function (plop: NodePlopAPI) {
         message:
           'The name of the security identity to impersonate, e.g. "alicesmith@example.com". See https://docs.coveo.com/en/56/#name-string-required.',
       },
-    ],
-    actions: [
       {
-        type: 'add',
-        path: currentPath + '/{{project}}/.env',
-        templateFile: '../template/.env.hbs',
-      },
-      {
-        type: 'addMany',
-        destination: currentPath + '/{{project}}/',
-        base: '../template',
-        templateFiles: [
-          '../template/src/**',
-          '../template/scripts/*',
-          '../template/.env.example',
-          '../template/tsconfig.json',
-          '../template/stencil.config.ts',
-          '../template/netlify.toml',
-          '../template/README.md',
-          '../template/start-netlify.mjs',
-        ],
-      },
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/.gitignore',
-        templateFile: '../template/.gitignore.hbs',
-      },
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/package.json',
-        templateFile: '../template/package.json.hbs',
-      },
-      function installPackagesPrompt() {
-        return 'Installing packages...';
+        type: 'input',
+        name: 'page-id',
+        default: '',
+        message:
+          '(Optional) The hosted search page ID, e.g. "7944ff4a-9943-4999-a3f6-3e81a7f6fb0a".',
```

Comment:
In terms of UX, how does one pick up its page ID?
(assuming the user go thru the Coveo CLI)

Reply:
Not very good at the moment, the page id is contained in the url. It would be easy to add a new table action in the admin to copy the id though!

Reply:
👍 imma adding that to the list of features that would benefit from a button to open the CLI.

Reply:
- [ ] Add prompt list to "pick your page"